.github/workflows/nodejs.yml

```diff
@@ -32,3 +32,28 @@ jobs:
         run: npm run lint
       - name: Test
         run: npm test
+
+  platform-compatibility-test:
+    name: Test platform compatibility
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macOS-latest, windows-latest]
+    env:
+      FORCE_COLOR: 1
+    steps:
+      - name: Checkout
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3
+      - name: Use Node.js
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3
+        with:
+          node-version-file: .nvmrc
```

Comment:
If we're going to keep this as its own job, can we just use a static version here?  `latest` or `lts` or whatever the default is.  I don't really want to add a build dependency upon `.nvmrc`.  I have mixed feelings about that file. :P